controllers/admin/AdminAjaxPrestashopCheckoutController.php

```diff
@@ -1003,4 +1003,25 @@ private function togglePayLaterConfiguration($method)
 
         $this->ajaxDie(json_encode(true));
     }
+
+    private function ajaxProcessUpsertSecretToken()
+    {
+        /** @var \PrestaShop\Module\PrestashopCheckout\Webhook\WebhookSecretTokenService $webhookSecretTokenService */
+        $webhookSecretTokenService = $this->module->getService('  ps_checkout.webhook.service.secret_token');
+
+        $secret = (string) Tools::getValue('body');
```

Comment:
```suggestion
        $secret = (string) Tools::getValue('body');
```
Is there a validation performed after ?

Reply:
No, I think the only validation I know right now would be to check if it's not empty - I can add that